src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -374,17 +452,12 @@ public override GroupBy GroupBy(int columnIndex, DataFrame parent)
             }
         }
 
-        public void ApplyElementwise(Func<T, T> func)
+        public void ApplyElementwise(Func<T?, long, T?> func)
```

Comment:
Is there a typical way to inform users that this function is not thread safe?

Reply:
Instance methods are never considered thread safe, unless they are explicitly stated to be.

---
In reply to: [315443932](https://github.com/dotnet/corefxlab/pull/2714#discussion_r315443932) [](ancestors = 315443932)